test/FixedPoint.spec.ts

```diff
@@ -356,15 +342,15 @@ describe('FixedPoint', () => {
       expect(await fixedPoint.getGasCostOfFraction(BigNumber.from(0), BigNumber.from(569))).to.eq(210)
     })
     it('gas cost of smaller numbers', async () => {
-      expect(await fixedPoint.getGasCostOfFraction(BigNumber.from(239), BigNumber.from(569))).to.eq(314)
+      expect(await fixedPoint.getGasCostOfFraction(BigNumber.from(239), BigNumber.from(569))).to.eq(588)
```

Comment:
oof that's pretty significant for a large portion of the input space

Reply:
yeah, not opposed to reverting this